src/types/smartSearch.ts

```diff
@@ -12,6 +12,7 @@ export enum FILTER_TYPE {
     SURVEY_OPTION='survey_option',
     SURVEY_RESPONSE = 'survey_response',
     SURVEY_SUBMISSION = 'survey_submission',
+    TASK = 'task',
```

Comment:
Here you're calling it `task`, but in the platform it seems to be called `tasks`, so I can't actually get the GUI to work with the platform API. Is it working for you?

Reply:
Hm, I believe I fixed that on the platform side?

Reply:
Ah yeah, the filter validator called it task but the service used tasks. It's been fixed now in zetkin-platform#433